web/src/lib/core/auth/providers/internal/AuthProviderClientSupplier.ts

```diff
@@ -1,9 +1,10 @@
-import { StitchAuthRoutes, StitchRequestClient } from "stitch-core";
+import { StitchAuthRoutes, StitchRequestClient, StitchAuthRequestClient } from "stitch-core";
 
-interface AuthProviderClientSupplier<T> {
+interface AuthProviderClientSupplier<ClientT> {
```

Comment:
[nit] RequestClientT

Reply:
This is actually not the request client type, but the type of Auth provider client that it returns, so I believe it's correct as is

Reply:
Yep. I thought this was the provider client signature. My bad